reflex/components/graphing/plotly.py

```diff
@@ -39,14 +43,22 @@ class Plotly(PlotlyLib):
     # If true, the graph will resize when the window is resized.
     use_resize_handler: Var[bool]
 
-    def _render(self) -> Tag:
```

Comment:
Turns out this wasn't actually doing anything... We can reinvestigate how to add this logic back in.